persistence/src/main/scala/hmda/persistence/processing/SubmissionFSM.scala

```diff
@@ -230,6 +230,15 @@ class SubmissionFSM(submissionId: SubmissionId)(implicit val domainEventClassTag
       val status = hmda.model.fi.Signed
```

Comment:
This block means that the FSM allows a submission to go from `ValidatedWithErrors` to `Signed` with the `Sign` event. (I added this back when we misunderstood how statuses worked.) Should this block be removed?